templates/component.tsx.hbs

```diff
@@ -0,0 +1,5 @@
+export interface {{pascalCase name}}Props {}
```

Comment:
Are we adopting interfaces for type declarations? because this is different from other MAAS repos

Reply:
The TS docs recommend using interfaces over types:

> ...we recommend you use interfaces over type aliases. Specifically, because you will get better error messages. If you hover over the following errors, you can see how TypeScript can provide terser and more focused messages when working with interfaces like Chicken.

Reply:
True, I just wanted to know if there's a different reason why we went with interfaces

Reply:
This is actually way more complex than it seems, recommendations are different depending on the use case, in particular when you're dealing with a component library.

https://twitter.com/mattpocockuk/status/1698642590193619025

You did raise a good point though, and I'm happy to stick with a `type` as a default if you prefer.